aptible/client_utils.go

```diff
@@ -4,6 +4,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
```

Comment:
might wanna move sort order simple enough and nitpick by me

Reply:
Yeah, my GoLand wasn't configured to group imports. Now it is and I updated all of the files I touched.